lib/modules/manager/ansible/extract.ts

```diff
@@ -2,17 +2,21 @@ import { logger } from '../../../logger';
 import { newlineRegex, regEx } from '../../../util/regex';
 import * as dockerVersioning from '../../versioning/docker';
 import { getDep } from '../dockerfile/extract';
-import type { PackageDependency, PackageFile } from '../types';
+import type { ExtractConfig, PackageDependency, PackageFile } from '../types';
 
-export function extractPackageFile(content: string): PackageFile | null {
+export function extractPackageFile(
+  content: string,
+  _filename = '',
+  config: ExtractConfig = {}
```

Comment:
```suggestion
  _filename,
  config: ExtractConfig 
```

no default values, fix tests